pandas/core/groupby/groupby.py

```diff
@@ -3232,9 +3232,12 @@ def first(
         self, numeric_only: bool = False, min_count: int = -1, skipna: bool = True
     ) -> NDFrameT:
         """
-        Compute the first entry of each column within each group.
+        Compute the first non-null entry of each column within each group.
```

Comment:
pandas documentation is quite consistent with using `NA` instead of `null`. Can you use `NA` throughout.

Reply:
In addition, this line is incorrect as you can pass `skipna=False`.